src/foam/dao/EasyDAO.js

```diff
@@ -201,13 +201,13 @@ if ( getOrder() != null &&
   }
 }
 
-if ( getAuthorize() || getAuthorizer() != null ) {
-  if ( getAuthorizer() != null ) delegate = new foam.nanos.auth.AuthorizationDAO(getX(), delegate, getAuthorizer(), getAuthorizeReads());
+if ( getAuthorizer() != null ) delegate = new foam.nanos.auth.AuthorizationDAO(getX(), delegate, getAuthorizer());
+else if ( getAuthorize() ) {
```

Comment:
I think you need to check for getAuthorize first and if thats true, then you proceed otherwise do nothing